santa.js

```diff
@@ -1,5 +1,23 @@
 function deliverPresents(houses) {
-  console.log('Delivering presents to ' + houses[0])
+  //Base case
+  //If we have a single house, deliver the presents and print delivered
+  if (houses.length === 0) return
+
+  if (houses.length === 1) {
+    console.log('Delivering presents to ' + houses[0])
+  } else {
+    //Break the Problem down
+    //Split the of more than one house into two lists
+
+    //Recall the function on a smaller piece
+    //Call deliver presents on both lists
+    //description of lists
+    var mid = parseInt(houses.length / 2)
+    var FirstHalf = houses.slice(0, mid)
```

Comment:
Remember that camelCase starts with all lowercase and capitalizes the first letter of words after the first. This should be `firstHalf` and `secondHalf`